loom/script/serialize/lsBinReader.cpp

```diff
@@ -308,6 +314,9 @@ MethodInfo *BinReader::readMethodInfo(Type *type)
     methodInfo->declaringType = type;
 
     readMethodBase(methodInfo);
+    if (methodInfo->getMissing()) {
+        type->setMissing("missing method %s", methodInfo->getName());
```

Comment:
This is only a partial message, so capitalization is ok. Full message would look something like
```
Ignoring missing type loom2d.math.Rectangle (missing method containsRect)
```